unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDController.cs

```diff
@@ -0,0 +1,95 @@
+namespace Legacy
+{
+    public class AvatarHUDController : IHUD
```

Comment:
Maybe we can take advantage of this opportunity to change the name to something less confusing, like `ProfileHud`, `IdentityToastHud` or whatever (not sure if this is in the scope of this PR, just general feedback).

Reply:
You are right, I will use the suggested names for the new HUD